File: demo-java/src/main/java/co/innoplayer/testapp/ClientApplication.java
package co.innoplayer.testapp;

import android.app.Application;
import android.content.Context;
import android.util.Log;

import androidx.multidex.MultiDex;

public class ClientApplication extends Application {
    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        try {
            MultiDex.install(this);
        } catch (RuntimeException e) {
            e.printStackTrace();
            Log.e("ClientApp", "printStacktrace: " + e.getMessage());
        }
    }
}

File: app/src/main/java/id/innovationcenter/lidoplayertest/ClientApplication.kt
package id.innovationcenter.lidoplayertest

import android.app.Application
import android.content.Context
import android.util.Log
import androidx.multidex.MultiDex
import androidx.multidex.MultiDexApplication

class ClientApplication : Application() {
    override fun attachBaseContext(base: Context?) {
        super.attachBaseContext(base)
        try {
            MultiDex.install(this)
        } catch (e : RuntimeException){
            e.printStackTrace()
            Log.e("ClientApp","printStacktrace: "+e.message)
        }
    }
    override fun onCreate() {
        super.onCreate()

    }
}